providers/base/units/power-management/jobs.pxu

```diff
@@ -161,10 +161,12 @@ requires: device.product == 'Lid Switch'
 _purpose:
     This test will check your lid sensor can detect lid close/open, and DUT will suspend when the lid closed
 _steps:
-    1. Press "Enter" to start the test
-    2. Close the lid (Please close the lid within 10 sec)
+    (Please wait at least 30 seconds after resuming from sleep before running this test. In most cases,
+     systemd will hold off on reacting to lid events in a period of time after system startup or resume.)
+    1. Press "Enter" to start the test.
+    2. Close the lid (Please close the lid within 10 sec).
     3. Wait 5 seconds with the lid closed.
```

Comment:
it's a bit confuse to me that the waiting time is not match in step 3 and the above notes.

Reply:
@stanley31huang Thanks for comment. They are different time, the time mentioned in the notes is the time should wait to run this test if system just resume from suspend. The time in step 3 is the time to close the lid. It's maybe really a bit confusing, I'm not sure how to make it a little more clear.

Reply:
It seems to me that we could simply write a scripts to check if current time is too close to previous suspend process.
If previous suspend process within 30 seconds, Ask user to wait a period of time and then close the lid.

Reply:
I think this is good idea. But I don't know how to get the how long the last suspend time to now. Do you have any idea about that? Thanks!

Reply:
If the scripts will make sure the system is ready for suspend test, we need to modify the steps here.